Makefile

```diff
@@ -52,8 +53,8 @@ container-push: container
 test: test-unit test-e2e
 
 .PHONY: test-unit
-test-unit:
-	@echo "FIXME: add unit tests"
+test-unit: build
```

Comment:
Why do you depend on `build` target in unit tests?

Reply:
Indeed I don't need it, I will change that. Thanks for catching that :)